streamlit_authenticator/validator.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+
+class Validator:
```

Comment:
Please follow the same commenting format as other classes and functions as shown below:

    def _check_credentials(self, inplace: bool=True) -> bool:
        """
        Checks the validity of the entered credentials.
        Parameters
        ----------
        inplace: bool
            Inplace setting, True: authentication status will be stored in session state, 
            False: authentication status will be returned as bool.
        Returns
        -------
        bool
            Validity of entered credentials.
        """